project.py

```diff
@@ -7,4 +7,11 @@
 current_time = datetime.datetime.now()
 
 # Printing value of now.
-print("Time now at greenwich meridian is:", current_time)
\ No newline at end of file
+print("Time now at greenwich meridian is:", current_time)
+
+#Printing out the distance between 19/07/23 and 01/01/24 
+from datetime import date
```

Comment:
By convention in python, imports are usually at the top of the file,

I'm new to python - [but from a quick google](https://stackoverflow.com/questions/128478/should-import-statements-always-be-at-the-top-of-a-module#:~:text=Imports%20are%20always%20put%20at,before%20module%20globals%20and%20constants.)

It looks like we can move them to the top and that this is normal

Reply:
Will fix as part of following PR